models/src/main/scala/coop/rchain/models/BlockHash.scala

```diff
@@ -10,4 +11,12 @@ object BlockHash {
   implicit class BlockHashOps(bs: BlockHash) {
     def base16String: String = Base16.encode(bs.toByteArray)
   }
+
+  implicit def ordering: Ordering[BlockHash] =
+    Ordering.by((b: ByteString) => b.toByteArray.toIterable)
+
+  implicit val show = new Show[BlockHash] {
+    def show(blockHash: BlockHash): String =
+      Base16.encode(blockHash.toByteArray).take(10)
+  }
```

Comment:
Define all three implicits for ByteString directly without alias, it will work for all defined aliases.

Also definition of _show_ is usually in representation that can be parsed and initial value can be constructed. So better to define it as an extension method for short print.